.github/pull_request_template.md

```diff
@@ -1,13 +1,24 @@
 # Objective
 
-- Describe the objective or issue this PR addresses.
-- If you're fixing a specific issue, say "Fixes #X".
+Describe the objective or issue this PR addresses.
 
-## Solution
+If you're fixing a specific issue, say "Fixes #X" and the linked issue will automatically be closed when this PR is merged.
```

Comment:
Maybe we make this part of it by default thing, like:

```
Fixes: #ISSUE_NUMBER
```

Reply:
I think that in practice these fixes comments are better handled inline with the objectives / test.

And we also don't have a guarantee that each PR has an associated issue.